client-js/app/scripts/app.ts

```diff
@@ -141,13 +141,20 @@ window.addEventListener('WebComponentsReady', () => {
 
   class RegistryActivity implements Activity {
     path: Array<string>;
+    private globElem: LabradRegistry;
```

Comment:
Using ```globElem``` is a hack. I tried making this ```private elem: LabradRegistry``` and switching the ```elem```s to ```this.elem``` like in ```DatavaultActivity``` but I get the following error

```
app/scripts/app.ts(158,14): error TS2322: Type 'Promise<{ elem: number; route: string; breadcrumbs: any[]; }>' is not assignable to type 'Promise<ActivityState>'.
  Type '{ elem: number; route: string; breadcrumbs: any[]; }' is not assignable to type 'ActivityState'.
    Types of property 'elem' are incompatible.
      Type 'number' is not assignable to type 'HTMLElement'.
        Property 'accessKey' is missing in type 'Number'.
app/scripts/app.ts(177,17): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type.
app/scripts/app.ts(177,22): error TS2304: Cannot find name 'elem'.
```